state/execution.go

```diff
@@ -236,6 +236,21 @@ func (blockExec *BlockExecutor) ApplyBlock(
 	return state, nil
 }
 
+func (blockExec *BlockExecutor) ExtendVote(height int64, round int32) (types.VoteExtension, error) {
+  ctx := context.Background()
```

Comment:
nit:
```suggestion
  ctx := context.TODO()
```

Do we want some form of cancellation, an app could hang here for what ever reason?